tests/integration_tests/test_sim.py

```diff
@@ -46,17 +47,17 @@ def set_dummy_sensor_values(sim_app, device_type: DeviceType, sensor_type: Senso
 class TestFlare:
     def test_arming(self, qtbot, sim_app, device_type):
         flush_packets(sim_app, device_type)
-        assert sim_app.rocket_data.last_value_by_device(device_type, DataEntryIds.STATE) == 0
+        assert sim_app.rocket_data.last_value_by_device(device_type, DataEntryIds.STATE) == STATE_IDS[0x00]
```

Comment:
Is there a reason this is better than just using `DataEntryValues.STATE_STANDBY` directly? If not, I think that might be cleaner. Also would address jesse's comment about magic number.

Reply:
Great point! Thanks